docs/pages/layouts/panel.md

```diff
@@ -16,7 +16,8 @@ Place patterns and interactions within panels on your pages to achieve focus and
 > {{ site.data.strings.headerDisclaimer }}
 
 ## Elements
-
+- `.fd-panel__header`: Panel header.
+- `.fd-panel__head`: Container for title and description.
 - `.fd-panel__title`: Title of the panel.
 - `.fd-panel__description`: Title of the description.
 - `.fd-panel__actions`: Panel level actions such as add, remove, delete, sort, etc.
```

Comment:
`fd-.panel__actions` still used?

Reply:
It is - we should have a followup to refine how flexbox is being used for the header, head, title, description and actions part of this. I think with some re-work a few of these could be eliminated - at least head and actions.